README.md

```diff
@@ -131,13 +131,19 @@ For instance, the age attribute uses the age converter, which calculates the age
 
 On the other hand, the ageRange attribute still uses the same birthDate base attribute, and a configuration object with an array of age ranges.
 
+## Extending the app and technical docs
+
+If you are interested in adding your service to the app, jump directly to [the extension tutorial](./docs/extending.md).
+
+If you want to know more about the inner technical workings of the app, proceed to the [docs folder](https://github.com/DECODEproject/decodev2/tree/master/docs).
+
 ## Implementation notes
 
 ### Redux store structure
 The store has the current state of the app.
 It contains as less information as possible, normalized for efficient access.
 
-The container components connect the store to the presentational components. It makes use of memoized selectors from the reselect library to retrieve data from the store and transform it to the shape that is more convenient to the presentational components.
+The container components connect the store to the presentational components. It makes use of memorized selectors from the reselect library to retrieve data from the store and transform it to the shape that is more convenient to the presentational components.
```

Comment:
It really is memoized, not memorized.

Reply:
Ups, this change I don't remember doing it..lol. Sorry =)